### PR TITLE
Fix: image manager

### LINKED ISSUE #3670: Crash when accessing the image manager
Crash when accessing the image manager by clicking on the front image

### PR DESCRIPTION
**Pull request by kartikaysharma01**
####  Description
The image manager launches and behaves fine,

#### Related issues

Fixes #3670 

#### Screenshots

fixi mage manager

### CODE REVIEW

**Comment by kartikaysharma01** [comment]
This fixes the image manager launch issue.
Another bug in image manager that i was not able to solve : Select Image not working
@VaiTon Please look at this and help me to solve that as well.

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/features/ImagesManageActivity.kt [hunk 1]
```diff
@@ -544,7 +544,7 @@ class ImagesManageActivity : BaseActivity() {
         }
     }
 
-    private val selectImageLauncher = registerForActivityResult(ImagesSelectActivity.Companion.SelectImageContract(binding.toolbar.title.toString()))
+    private val selectImageLauncher = registerForActivityResult(ImagesSelectActivity.Companion.SelectImageContract(""))
```
Comment (VaiTon) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/features/ImagesManageActivity.kt:547]: Why? That's the title of the chooser

**Comment by VaiTon** [comment]
> 
> 
> This fixes the image manager launch issue.
> Another bug in image manager that i was not able to solve : Select Image not working
> @VaiTon Please look at this and help me to solve that as well.

Could we open another issue on this? Do we have any log? @teolemon ?

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/features/ImagesManageActivity.kt (hunk 1 above)
Comment (kartikaysharma01) [reply app/src/main/java/openfoodfacts/github/scrachx/openfood/features/ImagesManageActivity.kt:547]: registerForActivityResult() is being called before onCreate() . Getting NPE as binding is null.

**Comment by kartikaysharma01** [comment]
> > This fixes the image manager launch issue.
> > Another bug in image manager that i was not able to solve : Select Image not working
> > @VaiTon Please look at this and help me to solve that as well.
> 
> Could we open another issue on this? Do we have any log? @teolemon ?

```
E/AndroidRuntime: FATAL EXCEPTION: main
    Process: openfoodfacts.github.scrachx.openfood.debug, PID: 16946
    android.content.ActivityNotFoundException: No Activity found to handle Intent { (has extras) }
        at android.app.Instrumentation.checkStartActivityResult(Instrumentation.java:2051)
        at android.app.Instrumentation.execStartActivity(Instrumentation.java:1709)
        at android.app.Activity.startActivityForResult(Activity.java:5192)
        at androidx.activity.ComponentActivity.startActivityForResult(ComponentActivity.java:584)
        at androidx.core.app.ActivityCompat.startActivityForResult(ActivityCompat.java:237)
        at androidx.activity.ComponentActivity$2.onLaunch(ComponentActivity.java:218)
        at androidx.activity.result.ActivityResultRegistry$2.launch(ActivityResultRegistry.java:162)
        at androidx.activity.result.ActivityResultLauncher.launch(ActivityResultLauncher.java:47)
        at openfoodfacts.github.scrachx.openfood.features.ImagesManageActivity.selectImage(ImagesManageActivity.kt:411)
        at openfoodfacts.github.scrachx.openfood.features.ImagesManageActivity.access$selectImage(ImagesManageActivity.kt:74)
        at openfoodfacts.github.scrachx.openfood.features.ImagesManageActivity$onCreate$3.onClick(ImagesManageActivity.kt:96)
        at android.view.View.performClick(View.java:7125)
        at android.view.View.performClickInternal(View.java:7102)
        at android.view.View.access$3500(View.java:801)
        at android.view.View$PerformClick.run(View.java:27336)
        at android.os.Handler.handleCallback(Handler.java:883)
        at android.os.Handler.dispatchMessage(Handler.java:100)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7356)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)

**Review by VaiTon — APPROVED** [review]

**Pull request merged**